### DISCUSSION TITLE
Curiosity about the game's name

### DISCUSSION THREAD

**Discussion by dugsean** [meikpiep/gauguin#63] [General]
First of all, thank you very much for publishing this game. I've just discovered it but am having a lot of fun with it.
I was just curious about why it's named after a painter.

**Comment by meikpiep** [comment]
Thanks for your feedback and your question.

A few months ago, the game was lacking two things: A (in my opinion) cool UI color theme and a name. I stumbled on https://colorlisa.com/ containing a list of artists, paintings and their primary colors. I simply probed a lot of them and the colors of Paul Gauguin's The Siesta matched immediately to me.

The search of the name of the app continued, but I was stuck to find a creative name that implies the kind of game just by its name. KenKen is under copyright, so I was sure to avoid this phrase to avoid fun meeting unknown lawyers. I realized that under this hypothesis (the name itself won't let to people install it), I have complete freedom in choosing the name. Thus, 'Gauguin' was the winner, thus the picture containing in the navigation menu.

There is neither deeper meaning, nor a deep connection between me and his paintings.
